plugin/bitrise/bitrise.go

```diff
@@ -6,16 +6,51 @@
 package bitrise
 
 import (
+	"github.com/joho/godotenv"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 )
 
+const (
+	envStoreFile = ".envstore.yml"
+)
+
 // Is returns true if the root path is a Bitrise
 // plugin repository.
-func Is(root string) bool {
-	path := filepath.Join(root, "step.yml")
+func Is(root, filename string) bool {
```

Comment:
can you explain this change?  The purpose of the `Is` function is to automatically determine if a codebase is Bitrise.  Provding the `step.yml` as an input should not be needed.

Reply:
I think this should be reverted and a new helper function should be created

Reply:
done